app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
I actually was stumped by this, so I'm going to copy/paste my notes.

Before with mongoid-enum:
```2.4.1 :001 > @user = User.first
 => #<User _id: BSON::ObjectId('5931ea4f2cde1152ba1ebcb5'), _role: :admin, call_order: ["5931ea512cde1152ba1ebcd1", "5931ea512cde1152ba1ebcd5", "5931ea512cde1152ba1ebccd", "5931ea512cde1152ba1ebcc9", "5931ea512cde1152ba1ebcc5", "5931ea512cde1152ba1ebcc1"], email: "test@test.com", line: nil, name: "testuser (admin)", patient_ids: [BSON::ObjectId('5931ea512cde1152ba1ebcc1'), BSON::ObjectId('5931ea512cde1152ba1ebcc5'), BSON::ObjectId('5931ea512cde1152ba1ebcc9'), BSON::ObjectId('5931ea512cde1152ba1ebccd'), BSON::ObjectId('5931ea512cde1152ba1ebcd5'), BSON::ObjectId('5931ea512cde1152ba1ebcd1')]>
```

with enumerize

```
=> #<User _id: BSON::ObjectId('5931ecbe2cde1156ca284844'), call_order: ["5931ecbf2cde1156ca284860", "5931ecbf2cde1156ca284864", "5931ecbf2cde1156ca28485c", "5931ecbf2cde1156ca284858", "5931ecbf2cde1156ca284854", "5931ecbf2cde1156ca284850"], email: "test@test.com", line: nil, name: "testuser (admin)", patient_ids: [BSON::ObjectId('5931ecbf2cde1156ca284850'), BSON::ObjectId('5931ecbf2cde1156ca284854'), BSON::ObjectId('5931ecbf2cde1156ca284858'), BSON::ObjectId('5931ecbf2cde1156ca28485c'), BSON::ObjectId('5931ecbf2cde1156ca284864'), BSON::ObjectId('5931ecbf2cde1156ca284860')], role: "admin”>
```

Long story short, mongoid uses symbols and aliases with _. enumerize explicitly states in the docs that it only deals with strings.